zipline/finance/trading.py

```diff
@@ -93,13 +92,34 @@ def __init__(
         max_date=None,
         env_trading_calendar=tradingcalendar
     ):
+        # `tc_tdays` is short for "trading calendar trading days"
+        # `tc_tday` is short for "trading calendar trading day"
+        tc_tdays = env_trading_calendar.trading_days
+        tc_tday = env_trading_calendar.trading_day
```

Comment:
tc_tday is used by loader()  in TradingEnvironment. Previously trading_day and trading_days for NYSE were hardcoded in loader.py but I suggested to pass them as a variables to loader().